.travis.yml

```diff
@@ -42,6 +42,10 @@ matrix:
         os: osx
         env: TEST=iOS
 
+      - osx_image: xcode10.2
+        os: osx
+        env: TEST=Unix
+
```

Comment:
@freak4pc We are already running tests on Xcode 9.3, 9.4,10.0 and 10.2.

This shouldn't be needed.

https://github.com/ReactiveX/RxSwift/blob/master/.travis.yml#L41-L43

Reply:
You can fix travis and add tests for Linux and 5.0. This is the test we don't have. Adding this won't add Linux tests.